packages/posix-bindings/posix-bindings.2.1.0/opam

```diff
@@ -0,0 +1,46 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "POSIX bindings"
+description: "install all available posix bindings"
+maintainer: ["romain.beauxis@gmail.com"]
+authors: ["Romain Beauxis"]
+license: "MIT"
+homepage: "https://github.com/savonet/ocaml-posix"
+bug-reports: "https://github.com/savonet/ocaml-posix/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ctypes"
```

Comment:
```suggestion
  "ctypes" {>= "0.14.0"}
```
uses `CArray.of_string`

Reply:
Applied upstream, directly to `posix-base` on which all packages depend. Thanks!